manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```diff
@@ -201,6 +201,9 @@ spec:
             postgresqlSharedBuffers:
               description: 'PostgreSQL shared buffers setting (default: 1GB)'
               type: string
+            storageClassName:
```

Comment:
Is this meant for all storage things, or just postgres?  If the latter we should probably prefix.

Reply:
All PVCs. I had a similar comment in https://github.com/ManageIQ/manageiq-pods/pull/511#discussion_r426686836.

We just only happen to have PVCs for PG and kafka/zookeeper.